app/helpers/storage_resource_helper/textual_summary.rb

```diff
@@ -28,10 +28,10 @@ def textual_name
   end
 
   def textual_logical_free
-    {:label => _("Used"), :value => @record.logical_free}
+    {:label => _("Free Space (GB)"), :value => @record.logical_free}
```

Comment:
Is this correct?  We should always be storing bytes in the database, and then formatting based on what the user chooses (particularly in reporting)

Reply:
fixed. see comment below